### PR TITLE
VZ-6766: Update doc with backup components

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ spec:
 | `prometheusOperator` | [PrometheusOperatorComponent](#prometheus-operator-component)     | The Prometheus Operator component configuration.   | No       |
 | `prometheusAdapter`  | [PrometheusAdapterComponent](#prometheus-adapter-component)       | The Prometheus Adapter component configuration.    | No       |
 | `kubeStateMetrics`   | [KubeStateMetricsComponent](#kube-state-metrics-component)        | The kube-state-metrics component configuration.    | No       |
+| `velero`             | [VeleroComponent](#velero-component)                              | The velero component configuration.                | No       |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:73]: The velero component configuration. -> The Velero component configuration.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 2]
```diff
@@ -70,6 +70,9 @@ spec:
...
+| `rancherBackup`      | [RancherBackupComponent](#rancher-backup-component)               | The rancher backup component configuration.        | No       |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:74]: The rancher backup component configuration. -> The rancherBackup component configuration.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 3]
```diff
@@ -361,3 +364,13 @@ To configure OpenSearch, instead of using install args, Oracle recommends that y
...
+
+### Velero Component
+| Field | Type | Description                             | Required
+| --- | --- |-----------------------------------------| --- |
+| `enabled` | Boolean | If true, then Velero will be installed. | No |
+
+### Rancher Backup Component
+| Field | Type | Description                                                                                                   | Required
+| --- | --- |---------------------------------------------------------------------------------------------------------------| --- |
+| `enabled` | Boolean | If true, then Rancher Backup will be installed. Rancher Backup is dependant on Rancher itself being installed | No |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:376]: then Rancher Backup will be installed. -> then rancherBackup will be installed.
Rancher Backup is dependant on Rancher itself being installed -> rancherBackup is dependent on Rancher being installed. (Please make all the corrections to this sentence, including a period at the end of the sentence.)

**Review by rosemarymarano — COMMENTED** [review]
Edits required.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 3 above)
Comment (theabmitra) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: done

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 2 above)
Comment (theabmitra) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: done

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 1 above)
Comment (theabmitra) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: done

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 4]
```diff
@@ -361,3 +364,13 @@ To configure OpenSearch, instead of using install args, Oracle recommends that y
...
+
+### Velero Component
+| Field | Type | Description                             | Required
+| --- | --- |-----------------------------------------| --- |
+| `enabled` | Boolean | If true, then Velero will be installed. | No |
+
+### Rancher Backup Component
+| Field | Type | Description                                                                                                  | Required
+| --- | --- |--------------------------------------------------------------------------------------------------------------| --- |
+| `enabled` | Boolean | If true, then rancherBackup will be installed. rancherBackup is dependant on Rancher itself being installed. | No |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:376]: You don't need the word "itself " in this sentence, "rancherBackup is dependent on Rancher being installed."
Please remove it.

**Review by rosemarymarano — APPROVED** [review]
One more fix, then it's fine. Thank you.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 4 above)
Comment (theabmitra) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: done

**Pull request merged**